src/utilities/io.cpp

```diff
@@ -294,10 +289,10 @@ write_gzipped_to_file(std::stringstream & ss, std::string const & file_name, boo
     std::exit(3);
   }
 
-  boost::iostreams::filtering_streambuf<boost::iostreams::input> out;
-  out.push(boost::iostreams::gzip_compressor());
-  out.push(ss);
-  boost::iostreams::copy(out, compressed);
+  seqan::VirtualStream<char, seqan::Output> filter;
+  seqan::open(filter, compressed, seqan::GZFile{});
+
+  filter << ss.rdbuf();
 }
```

Comment:
I am not sure this change is covered by the tests or the benchmark and it relies on a feature I didn't use before (directly streaming the read buffer to the other stream), so please double-check whether it doesn't break anything!

Reply:
Looks to me like it works as expected!